src/run_graph.rs

```diff
@@ -272,6 +273,10 @@ pub fn run_ex<DB: WriteResults + Sync>(
     threads_count: usize,
     config: &Config,
 ) -> Result<()> {
+    if !is_running() {
```

Comment:
I'd prefer if this was `if ::docker::is_running()`, so it's clear we're checking if docker is running.